Taskfile.yml

```diff
@@ -61,12 +56,28 @@ tasks:
       COGITO_TEST_REPO_NAME: '{{default "cogito-test-read-write" .COGITO_TEST_REPO_NAME}}'
       COGITO_TEST_REPO_OWNER: '{{default "pix4d" .COGITO_TEST_REPO_OWNER}}'
 
+  test:unit:
+    desc: Run the unit tests.
+    cmds:
+      - "{{.TESTRUNNER}} -count=1 -short -coverprofile=coverage.out ./..."
+
   test:integration:
     desc: Run the integration tests.
     cmds:
       - "{{.TESTRUNNER}} -count=1 -run 'Test.*Integration$' -coverprofile=coverage.out ./..."
     env: *test-env
 
+  test:all:
+    desc: Run all the tests (unit + integration). Use this target to get total coverage.
+    cmds:
+      - "{{.TESTRUNNER}} -count=1 -coverprofile=coverage.out ./..."
+    env: *test-env
+
+  browser:
+    desc: "Show code coverage in browser (usage: task test:<subtarget> browser)"
+    cmds:
+      - go tool cover -html=coverage.out
```

Comment:
@odormond 

> Can we get an ascii/ncurses rendering instead/too? :-D

Vscode integrates Go code coverage directly in the editor, it is quite neat.
I googled around and it seems that vim has the integration too: https://spacevim.org/use-vim-as-a-go-ide/#code-coverage
So one could argue that this target is not even really needed. I am forced to keep it because I am trying to go back to Sublime Text, which doesn't have Go code coverage integration :-D

On the other hand, I remember I have seen an ASCII rendering tool as you ask, so it should be easily doable. Let me know if you want me to investigate it more.

> More seriously, can this be used directly? I mean, I'm expecting to iterate between coding and running `task test:unit` until all is green. Then I'd like to just see the coverage without having to rerun the test, so using `task browser` (in that case, I think calling coverage would be clearer).

Running Go test is a different experience than other languages, they are faster than you typing `task test:unit` :-)
Seriously though, I think that what you want is already doable:

- iterate with `task test:unit`
- when you want to see the coverage, just type `task browser` instead of `task test:unit browser`.

Regarding the name, since we talked about ASCII rendering, a future-proof name seems to be `cover:browser` (which leaves space for `cover:ascii` or similar) ?

Reply:
Last point: note also that currently `task test:unit browser` will open the browser ONLY if the tests are successful.

Reply:
@odormond I am going to merge this one, feel free to reply anyway!

Reply:
Thanks @marco-m-pix4d! I will probably need to dig a bit into the vim integration but that sounds fairly easy to integrate.

About the `cover:browser`, I like the idea. Also I'm fine with just being able to run `task browser`. As it opens a browser anyway, the name is acceptable.